include/common/mpmc_queue.h

```diff
@@ -26,12 +26,10 @@
 
 namespace grppi{
 
-constexpr int DEFAULT_SIZE = 100;
-
 template <typename T>
 class mpmc_queue{
    private:
-      int size;
+      int size = 100;
```

Comment:
This is against "avoid magic constants" (http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es45-avoid-magic-constants-use-symbolic-constants)

Solution:

* Add a private data member constexpr static int default_size